### DISCUSSION TITLE
Add postup/postdown

### DISCUSSION THREAD

**Discussion by GaZaTu** [gravitl/netmaker#2143] [Ideas]
According to [v0.18.0](https://github.com/gravitl/netmaker/releases/tag/v0.18.0) `postup/postdown` has been removed.
I need the feature to assign a postup script on the client side to configure my firewall (firewalld on linux).
As such, i'd like the feature to be added again atleast on the client side. 😊

**Comment by mattkasun** [comment]
Enhanced firewall configuration is a planned feature but an implementation date has not been determined
